src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/common/ReasonMoneyTemplateData.java

```diff
@@ -94,7 +94,12 @@ public static ReasonMoneyTemplateData toReasonMoneyTemplateData(RecurringExpense
             .type(item.getType() == ExpenseTypeLRspec.OTHER
                       ? "Other: " + item.getTypeOtherDetails()
                       : EXPENSE_TYPE_LIP_RESPONSE.get(item.getType()))
-            .amountPounds((MonetaryConversions.penniesToPounds(item.getAmount())).setScale(2, RoundingMode.CEILING))
+            .amountPounds(getAmountPounds(item.getAmount()))
             .build();
     }
+
+    private static BigDecimal getAmountPounds(final BigDecimal item) {
+        return (item == null ? null : MonetaryConversions.penniesToPounds(item).setScale(2, RoundingMode.CEILING));
```

Comment:
if null then i would say set new BigDecimal(0) rather than null

Reply:
this is just utility class  and i would prefer 0 rather than null. I deally it should call this method itself if value is nulll

Reply:
That would make it look like the user had entered a value but they haven't